bathy_dave_plugin/launch/robots.launch

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<launch>
+
+    <!-- rexrov robot -->
+    <!-- <group ns="rexrov"> -->
+        <include file="$(find bathy_dave_plugin)/launch/rexrov.launch">
+            <arg name="robot_name" value="rexrov"/>
+            <arg name="init_pose" value="-x 485395.93 -y 58144.25 -z -1 -Y 3.14" />
+        </include>
```

Comment:
I made some minor changes to the initial position so we can move the ROV more quickly to a position that triggers the new tile.